GraphBulkImporter/Models/Asset.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GraphBulkImporter.Models
+{
+    class Asset
+    {
+        public int id { get; set; }
```

Comment:
If these classes in the Models folder are being used for serialization it would be better to use something like the DataMember attribute or JsonProperty (depending on how it is being serialized) rather than name the properties against convention

Reply:
Bonus is that you can use the attribute to specify if a member is required (not null) https://docs.microsoft.com/en-us/dotnet/api/system.runtime.serialization.datamemberattribute.isrequired?view=netframework-4.8#System_Runtime_Serialization_DataMemberAttribute_IsRequired I think you can also do it if you use the jsonpropertyattribute: https://www.newtonsoft.com/json/help/html/SerializationAttributes.htm#JsonPropertyAttribute 

Regarding not null, you can also use JsonConstructorAttribute to specify a constructor to use to deal with nonnullable members.

Reply:
This is fair. I wish i didn't have to decorate my class members with attributes just so that Cosmos DB is happy with id, not Id. I wish it would accept and use JsonSerializer settings to use camelCase as default. 

The new V3 SDKs do, but unfortunately the BulkExecutor hasn't moved, yet. 

I have changed the properties to be correct according to convention, and i deal with the case for Id and PartitionKey in the extension method itself.